src/app/models/Vehicle.ts

```diff
@@ -0,0 +1,12 @@
+export default interface Vehicle {
+  make: string;
+  model: string;
+  year: number;
+  miles: number;
+  color: string;
+  images: string[];
+  vin: string;
+  offerPrice: number;
+  sellingPrice: number;
+  seller: number;
+}
```

Comment:
Do we still need this?

Reply:
*this file

Reply:
there's a bug or something, the deletion of it wont commit